roles/st2web/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+---
+# tasks file for st2web
+
+- name: Add nginx key
+  sudo: yes
+  apt_key:
+    url: http://nginx.org/keys/nginx_signing.key
+    state: present
+
+- name: Add nginx repos
```

Comment:
Agreed. I was thinking about that myself, but I didn't know what was your policy on pulling in external dependencies, and I preferred to stay close to the way the setup was described in the docs.